pkg/assetfetcher/http_test.go

```diff
@@ -700,7 +702,7 @@ func TestMakeRequestWithAssetPayloadToAssetEventError(t *testing.T) {
 	defer ctrl.Finish()
 	mockRT := NewMockRoundTripper(ctrl)
 	resp := SiteAssetsResponse{
-		Resources: []Asset{{History: assetHistoryEvents{AssetHistory{Type: "SCAN", Date: "not a time"}}}},
+		Resources: []Asset{{History: assetHistoryEvents{AssetHistory{Type: "SCAN", Date: "1234-12-01T00:00:00Z"}}}},
```

Comment:
Question: same as above, why is this change necessary?

Reply:
I added a check before AssetPayloadtoAssetEvent() to validate the asset history, so in order to trigger AssetPayloadtoAssetEvent(), I needed to change the Date field to be something valid.